pkg/gce-pd-csi-driver/controller.go

```diff
@@ -332,7 +334,7 @@ func (gceCS *GCEControllerServer) CreateVolume(ctx context.Context, req *csi.Cre
 				if gce.IsGCEError(err, "notFound") {
 					return nil, status.Errorf(codes.NotFound, "CreateVolume source volume %s does not exist", volumeContentSourceVolumeID)
 				} else {
-					return nil, LoggedError("CreateVolume unknown get disk error when validating: ", err)
+					return nil, common.LoggedError("CreateVolume unknown get disk error when validating: ", err)
```

Comment:
this isn't your change, but what makes an error "unknown"? should we just say "failed to getDisk"?